privacy/readme/USAGE.rst

```diff
@@ -0,0 +1,2 @@
+In the "Data Protection", open the "Settings" menu to find and enable 
```

Comment:
Sorry, I don't see that as a significant improvement to the text.

Reply:
There's nothing to explain at the moment, really, because right now there are no configuration options available. This this should be replaced when new features are made available...